Makefile

```diff
@@ -16,49 +12,49 @@ lib-win: client-win swig compile-win
 
 .PHONY: client
 client: secrethub_wrapper.go
-	go build -o output/Client.a -buildmode=c-archive secrethub_wrapper.go
+	go build -o $(ODIR)/Client.a -buildmode=c-archive secrethub_wrapper.go
 
 .PHONY: client-win
 client-win: secrethub_wrapper.go
-	GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc go build -o output/Client.a -buildmode=c-archive secrethub_wrapper.go
+	GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc go build -o $(ODIR)/Client.a -buildmode=c-archive secrethub_wrapper.go
 
 .PHONY: swig
 swig:
-	$(SWIG) -csharp -namespace SecretHub $(ODIR)/secrethub.i
+	swig -csharp -namespace SecretHub $(ODIR)/secrethub.i
 
 .PHONY: compile
 compile: $(DEPS)
-	$(CC) -c -O2 -fpic -o $(ODIR)/secrethub_wrap.o $(ODIR)/secrethub_wrap.c
-	$(CC) -shared -fPIC $(OBJ) -o $(ODIR)/libClient.so
+	gcc -c -O2 -fpic -o $(ODIR)/secrethub_wrap.o $(ODIR)/secrethub_wrap.c
+	gcc -shared -fPIC $(OBJ) -o $(ODIR)/libClient.so
 
 .PHONY: compile-win
 compile-win: $(DEPS)
 	x86_64-w64-mingw32-gcc -c -O2 -fpic -o $(ODIR)/secrethub_wrap.o $(ODIR)/secrethub_wrap.c
 	x86_64-w64-mingw32-gcc -shared -fPIC $(OBJ) -o $(ODIR)/Client.dll
 
 .PHONY: dotnet-test
-dotnet: $(ODIR)/libClient.so
-	dotnet publish $(ODIR)/secrethub.csproj -o $(ODIR)
-	rm -r $(ODIR)/bin $(ODIR)/obj
-# 	dotnet $(ODIR)/secrethub.dll
-
-.PHONY: mono-test
-mono: $(ODIR)/libClient.so
-	mono-csc -out:$(ODIR)/runme.exe $(ODIR)/*.cs
-# 	mono ./$(ODIR)/runme.exe
+dotnet-test: $(ODIR)/libClient.so
+	dotnet publish $(ODIR)/secrethub.csproj -o $(ODIR)/build --nologo
+	mv $(ODIR)/libClient.so $(ODIR)/build 
+# 	dotnet $(ODIR)/build/secrethub.dll
```

Comment:
Shouldn't this be uncommented now that we have tests?

Reply:
Adjusted to a new version